JULIERAJ/KIDS-FIRST-Front
# Implement default page for messages section

## One Line Description
Description : This feature introduces a default page for the Messages section, displayed when user is not sharing a kid profile with anyone. 

## Test Steps
1. Open the application.
2. Navigate to the sidebar and click on messages
3.Verify that the default page is displayed with the correct prompt and navigation link.


## UI/UX Outcome
- [Insert relevant screenshots or videos here]
![image](https://github.com/user-attachments/assets/f6098cde-1868-410e-bc35-b6e04188589f)

## Checklist
